### PR TITLE
Feature/enhancements to the data request functionality

### PR DESCRIPTION
**Pull request by MarkCalvert**
https://salsadigital.atlassian.net/browse/DQL2-30
https://salsadigital.atlassian.net/browse/DQL2-31
https://salsadigital.atlassian.net/browse/DQL2-32

### CODE REVIEW

File: ckanext/data_qld/actions.py [hunk 1]
```diff
@@ -7,6 +7,7 @@
 import ckanext.datarequests.validator as validator
 import datetime
 import logging
+import ckan.lib.dictization.model_dictize as model_dictize
```
Comment (duttonw) [inline ckanext/data_qld/actions.py]: good to see library reuse 👍

**Review by duttonw — APPROVED** [review]

**Comment by duttonw** [comment]
can you enable circleci builds on your repo's (unsure if this is already done)

File: test/features/config.feature [hunk 2]
```diff
@@ -5,6 +5,5 @@ Feature: Config
         Given "Admin" as the persona
         When I log in
         And I visit "ckan-admin/config"
-        Then I should see "Data Request Config"
```
Comment (duttonw) [inline test/features/config.feature:8]: cucumber is given when then, with and's.  Please create a then answer for this test so the logic still works

File: ckanext/data_qld/actions.py [hunk 3]
```diff
@@ -91,6 +95,9 @@ def _undictize_datarequest_basic(data_request, data_dict):
     data_request.description = data_dict['description']
     organization = data_dict['organization_id']
     data_request.organization_id = organization if organization else None
+    if tk.h.closing_circumstances_enabled:
```
Comment (duttonw) [inline ckanext/data_qld/actions.py:102]: please make a function as you have this duplicated twice here and on line 335.

Comment (MarkCalvert) [reply ckanext/data_qld/actions.py:102]: Refactored into function

**Review by salsa-nathan — APPROVED** [review]

**Pull request merged**